### ISSUE TITLE
Addon doesn't affect `file://` URLs.

### ISSUE THREAD

**Issue by girst** [rNeomy/javascript-toggle-on-and-off#21]
I'm guessing this is a limitation of Firefox' WebExtension API?

(Thanks for this addon -- works flawlessly otherwise)

**Comment by krystian3w** [comment]
Yes, chrome / opera have option read local url on a extension list.
